### PR TITLE
Add example dags testcase as part of CI

### PR DESCRIPTION
**Pull request by utkarsharma2**
# Description
## What is the current behavior?
Add example dags as part of integration tests.

### CODE REVIEW

File: universal_transfer_operator/example_dags/example_dag_fivetran.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                    "skip_before": 0,
+                    "role_arn": "arn::your_role_arn",
+                    "file_type": "csv",
+                    "delimiter": "",
+                    "is_public": False,
+                    "on_error": "fail",
+                    "compression": "bz2",
+                    "table": "fivetran_test",
+                    "archive_pattern": "regex_pattern",
+                },
```
Comment (dimberman) [inline universal_transfer_operator/example_dags/example_dag_fivetran.py:65]: Might be easier to read to move this dictionary into its own variable so users that are looking at the DAG can focus on DAG structure.

File: universal_transfer_operator/example_dags/example_dag_fivetran.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                time_zone_offset="-5",
+                region="GCP_US_EAST4",
+                config={
+                    "host": "your-account.snowflakecomputing.com",
+                    "port": 443,
+                    "database": "fivetran",
+                    "auth": "PASSWORD",
+                    "user": "fivetran_user",
+                    "password": "123456",
+                },
```
Comment (dimberman) [inline universal_transfer_operator/example_dags/example_dag_fivetran.py:78]: Same here.

**Review by dimberman — APPROVED** [review]
Two minor comments but overal LGTM

File: universal_transfer_operator/example_dags/example_dag_fivetran.py (hunk 1 above)
Comment (utkarsharma2) [reply universal_transfer_operator/example_dags/example_dag_fivetran.py]: Done.

File: universal_transfer_operator/example_dags/example_dag_fivetran.py (hunk 2 above)
Comment (utkarsharma2) [reply universal_transfer_operator/example_dags/example_dag_fivetran.py]: Done.

**Pull request merged**